packages/controls/src/widget_bool.ts

```diff
@@ -101,6 +103,13 @@ class CheckboxView extends DescriptionView {
         this.label.style.display = indent ? '' : 'none';
     }
 
+    updateTabindex() {
+        if (!this.checkbox) return; // we might be constructing the parent
+        let tabindex = this.model.get('tabindex');
+        if (tabindex) this.checkbox.setAttribute('tabIndex', tabindex);
```

Comment:
Won't this `if (tabindex)` be false if `tabindex` is 0? Is this intended?

Reply:
@vidartf That's right. Comparing to `null` will be better (in case the original value is a non zero integer).